### PR TITLE
Js assignment

### PR DESCRIPTION
**Pull request by emtec-devyani-chaudhari**
Completed JS Assignment

### CODE REVIEW

File: JS-Assignment/assignment.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+
+
+
+// Create a function that takes the age in years and returns the age in days.
+//  calcAge(65) ➞ 23725
+//  calcAge(0) ➞ 0
+//  calcAge(20) ➞ 7300
+
+function calcAge(years) {
+    const days = years * 365
```
Comment (rishikesh-dhanawade-emtec) [inline JS-Assignment/assignment.js]: what if any of the year between is leap year, can we check condition for that?

**Review by rishikesh-dhanawade-emtec — COMMENTED** [review]
check comment

File: JS-Assignment/assignment.js (hunk 1 above)
Comment (emtec-devyani-chaudhari) [reply JS-Assignment/assignment.js]: Yes, We can
function calcAge(years) {
    const leapDays = parseInt(years / 4);
    console.log(leapDays)
    const days = (years * 365)+leapDays;
    console.log(`Age in days : ${days}`)
}

File: JS-Assignments/assignment.js [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+
+
+// Create a function that takes the age in years and returns the age in days.
+//  calcAge(65) ➞ 23725
+//  calcAge(0) ➞ 0
+//  calcAge(20) ➞ 7300
+
+/* function calcAge(years) {
+    const leapDays = parseInt(years / 4);
```
Comment (rishikesh-dhanawade-emtec) [inline JS-Assignments/assignment.js:74]: where is this logic?

**Review by rishikesh-dhanawade-emtec — CHANGES_REQUESTED** [review]
check comment

File: JS-Assignments/assignment.js (hunk 2 above)
Comment (emtec-devyani-chaudhari) [reply JS-Assignments/assignment.js:74]: Done

**Review by rishikesh-dhanawade-emtec — APPROVED** [review]
approved

**Pull request closed without merging**